sway-book/src/documentation/operations/assertions/assert.md

```diff
@@ -0,0 +1,13 @@
+# assert
+
+The `assert` function is automatically imported into every program from the [prelude](../../misc/prelude.md) and it takes an expression which must evaluate to a [boolean](../../language/built-ins/boolean.md). If the boolean is `true` then nothing will happen and the code will continue to run otherwise the virtual machine will revert.
```

Comment:
```suggestion
The `assert` function is automatically imported into every program from the [prelude](../../misc/prelude.md) and it takes an expression which must evaluate to a [Boolean](../../language/built-ins/boolean.md). If the Boolean is `true` then nothing will happen and the code will continue to run. Otherwise, the virtual machine will revert.
```